.travis.yml

```diff
@@ -11,16 +11,21 @@ filter_secrets: false
 os:
 - linux
 
+# language: node_js
+
+# node_js: node
```

Comment:
these commented lines can be deleted?

Reply:
Yes, didn't need that anymore after I changed it so it only installs Node.js when GUI=1